src/backend/seeds/06_feature_permissions.js

```diff
@@ -0,0 +1,33 @@
+const { ENV_IS_STAGING_OR_PROD } = require('../knexfile');
+
+if (!ENV_IS_STAGING_OR_PROD) {
+  exports.seed = function (knex) {
```

Comment:
The if-else statements both do the same thing, so we can remove the if statement.